lib/mayaUsd/ufe/UsdTRSUndoableCommandBase.cpp

```diff
@@ -43,23 +53,30 @@ void UsdTRSUndoableCommandBase<V>::initialize()
         addEmptyAttribute();
     }
 
-    // See
-    // https://stackoverflow.com/questions/17853212/using-shared-from-this-in-templated-classes
-    // for explanation of this->shared_from_this() in templated class.
     attribute().Get(&fPrevValue);
 }
 
 template<class V>
 void UsdTRSUndoableCommandBase<V>::undoImp()
 {
+    conditionalCreateItem();
+
     attribute().Set(fPrevValue);
     // Todo : We would want to remove the xformOp
     // (SD-06/07/2018) Haven't found a clean way to do it - would need to investigate
+
+    // Set fItem to nullptr because the command does not know what can go on with the prim inside
+    // its item after their own undo() or redo(). Setting it back to nullptr is safer because it means 
+    // that the next time the command is used, it will be forced to create a new item from the path, 
+    // or the command will crash on a null pointer.
+    fItem = nullptr;
```

Comment:
As per private conversation, this should go before line 62, at the beginning of undoImp().